### ISSUE TITLE
[Accessibility] Analyze button is very low contrast when disabled

### ISSUE THREAD

**Issue by PeterSaathoff-Harshfield** [ampproject/amp.dev#4487]
Environment:
Any OS/browser

Steps to reproduce:
1. Open https://amp-dev-staging.appspot.com/page-experience/
2. before entering a URL, observe Analyze button

The current contrast ratio of the text on this button is 1.23:1

The text/contrast ratio should be at least 4.5:1 for most elements. See [WCAG 1.4.3](https://www.w3.org/TR/WCAG21/#contrast-minimum) and [Understanding Success Criterion 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

The button is in the disabled state, so lower contrast makes sense, and the text is 16pt, but I would recommend a ratio of at least 3:1.
![image](https://user-images.githubusercontent.com/30604285/92810420-62f0f000-f372-11ea-9bd1-45a56e949bcc.png)
